build.gradle

```diff
@@ -53,6 +53,38 @@ publishing {
             }
             artifactId distroType
             version "${jdk_version}-${jdk_build}"
+            pom {
+                licenses {
+                    // official SPDX identifier
+                    // see https://spdx.org/licenses/ for list
+                    license {
+                        name = "GPL-2.0-only"
+                        url = "https://github.com/JetBrains/JetBrainsRuntime/blob/jb${jdk_version}-${jdk_build}/LICENSE"
+                        comments = "GNU GENERAL PUBLIC LICENSE Version 2, June 1991"
+                        distribution = "repo"
+                    }
+                    license {
+                        name = "WITH Classpath-exception-2.0"
+                        url = "https://github.com/JetBrains/JetBrainsRuntime/blob/jb${jdk_version}-${jdk_build}/ADDITIONAL_LICENSE"
```

Comment:
The file is called [ADDITIONAL\_LICENSE\_INFO](https://github.com/JetBrains/JetBrainsRuntime/blob/main/ADDITIONAL_LICENSE_INFO).

Reply:
thank you for spotting the issue, fixed